dev/devicelab/bin/tasks/module_test_ios.dart

```diff
@@ -439,10 +439,15 @@ end
           'Host.app',
         );
 
-        checkFileExists(path.join(
+        final String builtHostBinary = path.join(
           archivedAppPath,
           'Host',
-        ));
+        );
+        checkFileExists(builtHostBinary);
+        if ((await fileType(builtHostBinary)).contains('armv7')) {
+          throw TaskResult.failure('Unexpected armv7 architecture slice in $builtHostBinary');
+        }
+        await containsBitcode(builtHostBinary);
```

Comment:
what does this do?

Reply:
You're right, this is a bug.  Will fix all the spots in this file.